### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by tomlcb888** [Chia-Network/chia-blockchain#5007] [Support]
Where is my coin transferred to? I used the same mnemonic to create the wallet on my laptop and desktop computer respectively. After opening the transaction function on May 4th, I transferred the coin to my laptop wallet through the mine, but I haven't received it yet. But the chain has confirmed (wallet address: xch193jyh24f8ve07w8wuzzgra89ewndl7lrjyfc834xgfp0f4yjfv2sqvrhy5), please find. Thank you very much!

**Comment by lvpcguru** [comment]
If you used the same mnemonic you should have the exact same private keys on both machines. There should not be a second wallet at all. The blockchain shows transfer you made to xch193jyh24f8ve07w8wuzzgra89ewndl7lrjyfc834xgfp0f4yjfv2sqvrhy5, are you sure you typed this one correctly? that transfer comes as a small portion of a very large amount that didnt come from a farm.

[https://www.chiaexplorer.com/blockchain/address/xch193jyh24f8ve07w8wuzzgra89ewndl7lrjyfc834xgfp0f4yjfv2sqvrhy5](https://www.chiaexplorer.com/blockchain/address/xch193jyh24f8ve07w8wuzzgra89ewndl7lrjyfc834xgfp0f4yjfv2sqvrhy5)

**Comment by tomlcb888** [comment]
I can make sure that the address is through the copy button of my wallet, copy and paste it and check the last few digits. It is only possible that the wallet has not been updated to the latest state at that time. Maybe it was before May.

------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Chia-Network/chia-blockchain"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年5月26日(星期三) 凌晨1:39
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Chia-Network/chia-blockchain] [BUG] (#5007)





 
If you used the same mnemonic you should have the exact same private keys on both machines. There should not be a second wallet at all. The blockchain shows transfer you made to xch193jyh24f8ve07w8wuzzgra89ewndl7lrjyfc834xgfp0f4yjfv2sqvrhy5, are you sure you typed this one correctly? that transfer comes as a small portion of a very large amount that didnt come from a farm.
 
https://www.chiaexplorer.com/blockchain/address/xch193jyh24f8ve07w8wuzzgra89ewndl7lrjyfc834xgfp0f4yjfv2sqvrhy5

**Comment by BrandtH22** [comment]
@tomlcb888 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
